liquidhaskell-boot/src/Language/Haskell/Liquid/Bare/Expand.hs

```diff
@@ -410,7 +420,7 @@ expandBareType rtEnv _ = go
     go t@RHole{}         = t
     go t@RVar{}          = t
     go t@RExprArg{}      = t
-    goRef (RProp ss t)   = RProp ss (go t)
+    goRef (RProp ss t)   = RProp (map (expand rtEnv l <$>) ss) (go t)
```

Comment:
Summoning @nikivazou, in case there is a good reason why types in `ss` were not expanded.

Reply:
No reason that I can remember!